webapp/components/test-search.js

```diff
@@ -39,9 +39,20 @@ const QUERY_GRAMMAR = ohm.grammar(`
     Q = ListOf<RootExp, space*>
 
     RootExp
-      = "seq(" ListOf<Exp, space*> ")" -- sequential
+      = Sequential
+      | Count
```

Comment:
count operator exists at the root level?

Reply:
We shouldn't be combining it with other queries; you'd get unexpected behaviour